neurom/features/__init__.py

```diff
@@ -141,10 +81,14 @@ def get(feature, obj, **kwargs):
     Returns:
         features as a 1D or 2D numpy array.
     """
-    feature = (NEURITEFEATURES[feature] if feature in NEURITEFEATURES
-               else NEURONFEATURES[feature])
+    for feature_list in (NEURITEFEATURES, NEURONFEATURES):
```

Comment:
`NEURITEFEATURES, NEURONFEATURES` are dicts and not lists, so I would not use the suffix `_list` in the local variable `feature_list` because it can be misleading (the same could be applied to other parts of the code using this same variable name).